GeorgiaTech/Api/Models/Member.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+
+namespace Api.Models
+{
+    public partial class Member
+    {
+        public Member()
+        {
+        }
+
+        public int MemberId { get; set; }
+        public string SSN { get; set; }
+        public string FName { get; set; }
+        public string LName { get; set; }
+        
+
+        public virtual ICollection<string> PhoneNumbers { get; set; }
+        public virtual string CampusAddress { get; set; }
```

Comment:
You could also just reference the `Api.Models.Address` class for both CampusAddress and HomeAddress.

Reply:
Gonna refactor it to that then - no reason not to use them. 
- Gonna clean up the API endpoints so it doesn't conflict.